Sources/Build/BuildPlan.swift

```diff
@@ -575,6 +592,37 @@ public final class SwiftTargetBuildDescription {
         return args
     }
 
+    /// Returns true if ObjC header should be emitted.
+    private var shouldEmitObjcHeader: Bool {
```

Comment:
Maybe we should call this `shouldEmitObjCCompatibilityHeader` to make it more clear what this is for?